ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -22,6 +22,7 @@ struct MSFButtonViewButtonStyle: ButtonStyle {
         let state = targetButton.state
         let isDisabled = state.isDisabled
         let isPressed = configuration.isPressed
+        let isActive = isDisabled || isPressed
```

Comment:
why is isActive = isDisabled or isPressed?

Reply:
The button has a smaller shadow when it's in a state other than rest. I thought "isActive" was a bit better than "isNotAtRest". But there's probably a better option.

Reply:
if this boolean variable is about shadow than the variable name should be more geared towards shawdow? either state enum has func like shouldShowSmallShadow()->Bool or something that will give more context how this variable is needed?

Reply:
isDisabled is coming from the state, which isPressed is coming from the configuration, so adding the function to the state enum probably wouldn't work. But I did change the name.